foreman/foreman.spec

```diff
@@ -107,8 +107,8 @@ Requires: %{?scl_prefix}rubygem(rails-observers) >= 0.1
 Requires: %{?scl_prefix}rubygem(rails-observers) < 1.0
 Requires: %{?scl_prefix}rubygem(protected_attributes) >= 1.1.1
 Requires: %{?scl_prefix}rubygem(protected_attributes) < 1.2.0
-Requires: %{?scl_prefix_ror}rubygem(sprockets-rails) >= 2.2.2
-Requires: %{?scl_prefix_ror}rubygem(sprockets-rails) < 3
+Requires: %{?scl_prefix}rubygem(sprockets-rails) >= 2.2.2
```

Comment:
Yup, that one changes it. I think I changed this when I was building the whole mockchain and this package needed to rely on the tfm sprockets-rails.